build.gradle

```diff
@@ -37,35 +29,34 @@ dependencies {
 
   //Gradle doesn't yet handle dependency resolution conflicts for localGroovy()
   //See: http://forums.gradle.org/gradle/topics/conflicting_module_versions_with_groovy_gradle_2_0_rc_1
-  testCompile('com.netflix.nebula:nebula-test:2.2.0') {
+  testCompile('com.netflix.nebula:nebula-test:2.2.1') {
     exclude group:'org.codehaus.groovy'
   }
 
   //Allow testing of optional plugin configuration
   testCompile (
     'org.jfrog.buildinfo:build-info-extractor-gradle:3.0.1',
-    'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.0',
+    'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2',
+    'com.gradle.publish:plugin-publish-plugin:0.9.0',
   )
 }
 
-test {
-  maxParallelForks = 2
+tasks.withType(Test) {
+  maxParallelForks = Runtime.runtime.availableProcessors()
```

Comment:
OK. I'll add a note in the README on that as well. If it causes problems for people, then they can override it in the appropriate tasks. I do think it may be more problematic in `integTest` and `e2e` tests than in unit tests - do we just want to do it for the standard unit test tasks for now?